tests/basetest.rb

```diff
@@ -133,7 +133,6 @@ def setup
   def teardown
     @device.close unless @device.nil?
     @device = nil
-    GC.start
```

Comment:
I'm not sure it's actually needed for one (this has been in the code base since the OneP days, IIRC), and I was intermittently seeing an error (when trying to run the whole suite) about allocation during garbage collection which caused the entire test process to abort. This error seemed to go away after I removed this line.